osm2gtfs/tests/creators/creators_tests.py

```diff
@@ -0,0 +1,368 @@
+import unittest
+import os
+import zipfile
+import csv
+import overpy
+import transitfeed
+
+from mock import patch
+from osm2gtfs.core.configuration import Configuration
+from osm2gtfs.core.osm_connector import OsmConnector
+from osm2gtfs.core.creator_factory import CreatorFactory
+from osm2gtfs.core.cache import Cache
+
+current_dir = os.path.dirname(__file__)
+
+
+class CreatorsTestsArgs():
+    def __init__(self, config_file, selector):
+        """
+        Prepare arguments for the Initialization of an Configuration object
+        (see osm2gtfs.core.configuration for more information)
+
+        """
+        # Basic preparations
+        self.config = open(config_file)
+        self.selector = selector
+
+        # Overriding the output to not interfere with user's data
+        self.output = os.path.realpath(
+            os.path.join(current_dir, "../../../data/" + self.selector + "_tests.zip"))
+
+
+class CreatorsTestsAbstract(unittest.TestCase):
+
+    def _get_selector(self):
+        """
+        Needs to be implemented for providers. And return the selector stroing.
+        """
+        raise NotImplementedError
+
+    def _get_required_values(self):
+        """
+        Needs to be implemented for providers. And return the required values.
+        A dictionary containing the following keys is expected to be returned:
+
+        * routes_count
+        * stops_count
+        * stations_count
+        * stops_osm_count
+        * route_id_to_check
+        * gtfs_files
+
+        """
+        raise NotImplementedError
+
+    def _get_standard_values(self):
```

Comment:
I don't like `values`. Maybe `variables` or `test_config` ?

Reply:
Sure, I can rename it to `variables`.